frontend/src/pages/LandlordPage.tsx

```diff
@@ -80,10 +81,18 @@ const LandlordPage = (): ReactElement => {
     });
   }, []);
 
+  // re-renders when the data changes
```

Comment:
Oh that's a really good idea! This whole time I was thinking of keeping the reviewData state sorted, which is why I was using 2 different useEffects because I needed to destructure the array if the array is the same but sorted differently, but that's actually not necessary with your suggestion. I tested your suggestion and it works!